plugins/events-node/README.md

```diff
@@ -1,3 +1,88 @@
-# plugin-events-node
+# `@backstage/plugin-events-node`
 
-Houses types and utilities for building events-related modules.
+This package defined basic types for event-based interactions inside of Backstage.
+
+Additionally, it provides the core event service `eventServiceRef` of type `EventService`
+as well as its default factory implementation `eventServiceFactory`
+that uses the `DefaultEventService` implementation.
+
+`DefaultEventService` is a simple in-memory implementation
+that requires the co-deployment of producers and consumers of events.
+
+## Installation
+
+```bash
+# From your Backstage root directory
+yarn add --cwd packages/backend @backstage/plugin-events-node
```

Comment:
I guess I would still need to add the package, right?

Reply:
No, just the act of installing a plugin who in turn depends on the service ref ensures that we "discover" the default factory.

Reply:
yes, that makes sense. It would be pulled in as dependency (if it has a default implementation), and otherwise, it is not needed anyway.

I took care of a couple of other points first and plan to take care of the service and its factory later today.